### DISCUSSION TITLE
Can I use  p2-es server-side?

### DISCUSSION THREAD

**Discussion by rev111** [pmndrs/p2-es#70] [General]
Hello,
I see I can import a node module, but can I run p2-es in a way to do the physics calculations owith node on the server without rendering them?
Because I saw that p2-es uses glmatrix for calculations and I wonder if that would work on a server with no or limited GL support...

**Comment by isaac-mason** [comment]
There's nothing stopping you from using p2-es in node or other server environments, it's all just javascript 🙂

If you're interested, this is the "vec2" functionality from glmatrix: https://github.com/pmndrs/p2-es/blob/master/packages/p2-es/src/math/vec2.ts
